.github/workflows/release.yml

```diff
@@ -14,12 +14,6 @@ jobs:
     steps:
       - name: Checkout the repository
         uses: actions/checkout@v2
-      - uses: satackey/action-docker-layer-caching@v0.0.11
```

Comment:
This was the old way of doing things but should have been removed as a part of 784f7ac6062cda45a3717b2278e5f1241f18f5ea where we moved the info into the `tasks.py` "build" task ([here](https://github.com/SeisoLLC/easy_infra/blob/60996376256524f5fd65941ced1e2eab21ede366/tasks.py#L273))